### DISCUSSION TITLE
Professional Support for PyVista

### DISCUSSION THREAD

**Discussion by banesullivan** [pyvista/pyvista#4033] [Announcements]
### _Need help getting the most out of PyVista?_  or _Want to sponsor development of new capabilities?_

Reach out to [info@pyvista.org](mailto:info@pyvista.org) and we can help connect you with the right community experts to tackle your challenges.

You can also reach out to individual contributors for sponsorship/help:

- @banesullivan: https://github.com/sponsors/banesullivan

------

### Additional organizations providing services

[Kitware](https://www.kitware.com/) provides professional services for PyVista, including custom solution creation, collaborative research and development, custom support, and training.

Please reach out to [kitware@kitware.com](mailto:kitware@kitware.com) to connect with their expert R&D team, who can help you confidently and efficiently use PyVista and Trame and/or tailor your workflows to leverage PyVista and VTK’s robust 3D visualization capabilities.

**Comment by BobHu14** [comment]
Can I run pyvista on GPU？ I need to render arrow functions, but I have hundreds of thousands of data points. How can I render them quickly?
